<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SetROI</class>
 <widget class="QWidget" name="SetROI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>961</width>
    <height>730</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <property name="horizontalSpacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <property name="spacing">
      <number>8</number>
     </property>
     <item>
      <widget class="ChildTitleBar" name="widget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>30</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>30</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout" stretch="2,1,2">
         <item>
          <widget class="QTableWidget" name="tableWidget">
           <property name="dragEnabled">
            <bool>true</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <row>
            <property name="text">
             <string>雷达Ip</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>雷达序号</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>雷达扫描半径</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>可编辑区域序号</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>基准坐标</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>基准转角</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>像素距离比</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>区域内点数</string>
            </property>
           </row>
           <row>
            <property name="text">
             <string>区域外点数</string>
            </property>
           </row>
           <column>
            <property name="text">
             <string>属性</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="frame_edit">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QGridLayout" name="gridLayout_2">
            <property name="leftMargin">
             <number>2</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>2</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item row="0" column="1">
             <widget class="QComboBox" name="comboBox">
              <item>
               <property name="text">
                <string>1</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>2</string>
               </property>
              </item>
              <item>
               <property name="text">
                <string>3</string>
               </property>
              </item>
             </widget>
            </item>
            <item row="0" column="0">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>区域选择</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0">
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>防区高度最大值（单位：m）</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QLineEdit" name="lineEdit_times"/>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>防区高度最小值（单位：m）</string>
              </property>
             </widget>
            </item>
            <item row="3" column="1">
             <widget class="QLineEdit" name="lineEdit_Area_height_max"/>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>障碍物存在时间</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QLineEdit" name="lineEdit_Area_height_min"/>
            </item>
            <item row="4" column="1">
             <widget class="QLineEdit" name="lineEdit">
              <property name="text">
               <string>400</string>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>防区半径（单位：m）</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tableWidget_2">
           <property name="dragEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <column>
            <property name="text">
             <string>X</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>Y</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ChildTitleBar</class>
   <extends>QWidget</extends>
   <header location="global">childtitlebar.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
